src/renderer/src/routes/pages/Send/SendPageContent.tsx

```diff
@@ -363,7 +364,9 @@ export const SendPageContent = ({ account, recipientAddress }: TProps) => {
         let totalFeeAmount = NumberHelper.number(fee)
 
         fields.intents.forEach(intent => {
-          if (UtilsHelper.normalizeHash(intent.tokenHash) !== UtilsHelper.normalizeHash(fields.service.feeToken.hash))
+          if (
+            BSTokenHelper.normalizeHash(intent.tokenHash) !== BSTokenHelper.normalizeHash(fields.service.feeToken.hash)
```

Comment:
Create a variable for this value and reuse it: `BSTokenHelper.normalizeHash(fields.service.feeToken.hash)`.